### ISSUE TITLE
Support laravel 8

### ISSUE THREAD

**Issue by MarJose123** [phpsa/filament-password-reveal#10]
https://github.com/phpsa/filament-password-reveal/blob/a1617d164bc947d49581578c59aa581f10082da1/src/Password.php#L62

Maybe we can have a condition like this
```php
if(app()->version() == 8.0.0){
        return Str::of($this->getId())->replace(".", "_")->prepend('input_')->studly()->snake()->toHtmlString();
}
        return Str::of($this->getId())->replace(".", "_")->prepend('input_')->studly()->snake()->toString();
```

**Comment by phpsa** [comment]
Would accept a PR for this

**Comment by phpsa** [comment]
Actually fixed in 1.1.3
